build/compile/Microsoft.DotNet.Cli.LzmaArchive.targets

```diff
@@ -72,7 +72,12 @@
       <MakeDir Directories="@(NuGetPackagesArchiveDirectory)"/>
 
       <DotNetNew ToolPath="$(Stage2Directory)"
-                 TemplateType="Web"
+                 TemplateType="--debug:reinit"
```

Comment:
We added a second parameter to make the additional arguments more clear - this item in particular though has been removed in the commit I'm testing locally now.